src/Etg.Yams.Client/YamsClient.cs

```diff
@@ -16,9 +17,10 @@ public class YamsClient : IYamsClient
 
         public event EventHandler ExitMessageReceived;
 
-        public YamsClient(YamsClientConfig config, IIpcConnection initConnection, IIpcConnection exitConnection,
+        internal YamsClient(YamsClientOptions options, YamsClientConfig config, IIpcConnection initConnection, IIpcConnection exitConnection,
```

Comment:
Why internal? I have no issues with people using this class directly if they wish to do so :)

Reply:
I though the factory is supposed to be used in all times. Rolling back